srcpkgs/qodem/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'qodem'
+pkgname=qodem
+version=1.0.1
+revision=1
+build_style=gnu-configure
+makedepends="ncurses-devel gpm-devel"
+depends="libgpm"
```

Comment:
this should be automatically detected

Reply:
Thanks, I wasn't sure if it was best practice to let xbps detect dependencies or not.